tutorial/server/axum/assets/auth.js

```diff
@@ -0,0 +1,93 @@
+function register () {
+    let username = document.getElementById('username').value;
+    if (username === "") {
+        alert("Please enter a username");
+        return;
+    }
+
+    fetch('http://localhost:8080/register_start/' + username, {
```

Comment:
`username` [needs to be escaped](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) here, and also below.

(it also looks like this is a bug with the Rust/WASM bindings, but that doesn't need to be addressed in this PR)